src/main/java/com/glovo/ttglovo/cart/CartItem.java

```diff
@@ -0,0 +1,59 @@
+package com.glovo.ttglovo.cart;
+
+import com.glovo.ttglovo.securityManagement.appuser.AppUser;
+import lombok.*;
+
+import javax.persistence.*;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
+@Entity(name = "CartItem")
+@Table(name = "cart_items")
+public class CartItem {
+
+    @Id
+    @SequenceGenerator(
+            name = "cart_sequence",
+            sequenceName = "cart_sequence",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "cart_sequence"
+    )
```

Comment:
This can be simplified with a GenerationType.IDENTITY strategy. You don't need to worry about specifying the sequence in that case.